### PR TITLE
Collecting Reports

### PR DESCRIPTION
**Pull request by ccolliga**
I added a Collecting Reports page. It is still drafty.

### CODE REVIEW

File: docs/collecting_reports.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+## Illustrations 
+
+Collect illustrations of Wilde, the trials, and his imprisonment from the relevant newspaper articles for inclusion in the archive's Image Gallery. 
+
+
+## Newspaper Collection Tools
+
+For Australian newspapers in the Trove database, use the custome-built Trove tool to collect articles. http://www.sfu.ca/~mjoyce/wt/
+Enter the API key (mib19ga3a20n24tt) and the individual article ID
```
Comment (ubermichael) [inline docs/collecting_reports.md]: This API key should not be public. I suggest "Contact the PI, developers, or DHIL for the correct API key."

Comment (ccolliga) [reply docs/collecting_reports.md]: Done. I've added two pages to the index and made revisions throughout. Let me know what you think, and make changes as you see fit. Once you have accepted the PR, I'll have my RAs look it over to see if they want to add anything.

File: docs/collecting_reports.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+For headlines, make sure to include the title of the column (if there is one) as well as the article headline and sub-headlines. 
+
+## Illustrations 
+
+Collect illustrations of Wilde, the trials, and his imprisonment from the relevant newspaper articles for inclusion in the archive's Image Gallery. 
+
+
+## Newspaper Collection Tools
+
+For Australian newspapers in the Trove database, use the custome-built Trove tool to collect articles. http://www.sfu.ca/~mjoyce/wt/. Copy and paste the marked up text into a new file in Oxygen for editing.
```
Comment (ubermichael) [inline docs/collecting_reports.md]: Is custome a typo?

File: docs/collecting_reports.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+## Collection Protocols
+To determine what constitutes an article, prioritize evidence from the newspaper over the database’s article segmentation. In other words, does the newspaper represent it as an individual article? (eg. Does it have a headline; is it separated from other articles?). In most cases, it should be easy to figure out what is an individual news article.
+
+If there are a few different news articles published on the same day which appear in different parts of the newspaper, each of these news articles is considered a distinctive news article and gets transcribed as an individual file.
+
+If there is one long column, with multiple news items, including items pertaining to the Wilde trials, transcribe the relevant news items, while omitting the non-relevant news items. Use [xyz] to indicate if there is text that you have excerpted in the middle of the selected news artcle.
```
Comment (ubermichael) [inline docs/collecting_reports.md]: Is [xyz] the correct substitution? I thought we were using `<p class='omitted'></p>` or something similar.

**Comment by ccolliga** [comment]
Fixed


Dr. Colette Colligan, Professor of English

Simon Fraser University, Canada




________________________________
De : Michael Joyce ***@***.***>
Envoyé : dimanche 18 juillet 2021 03:17
À : sfu-dhil/wilde-schema
Cc : Colette Colligan; Author
Objet : Re: [sfu-dhil/wilde-schema] Collecting Reports (#1)


@ubermichael commented on this pull request.

________________________________

In docs/collecting_reports.md<https://github.com/sfu-dhil/wilde-schema/pull/1#discussion_r671762722>:

> +
+In cases where there are two articles that appear one after the other, some judgement call will be needed. If a significant headline or large dividing feature separates the two items, treat them as separate articles and transcribe each in its own separate file.
+
+Do not collect articles that make only minor reference to Wilde and are not directly related to the trials and his imprisonment.
+
+## Headlines
+For headlines, make sure to include the title of the column (if there is one) as well as the article headline and sub-headlines.
+
+## Illustrations
+
+Collect illustrations of Wilde, the trials, and his imprisonment from the relevant newspaper articles for inclusion in the archive's Image Gallery.
+
+
+## Newspaper Collection Tools
+
+For Australian newspapers in the Trove database, use the custome-built Trove tool to collect articles. http://www.sfu.ca/~mjoyce/wt/. Copy and paste the marked up text into a new file in Oxygen for editing.


Is custome a typo?

**Comment by ccolliga** [comment]
Yes, I will add that now. The only problem is I don't yet know what mark I should use to indicate omitted text in the middle of a long colum.


Dr. Colette Colligan, Professor of English

Simon Fraser University, Canada




________________________________
De : Michael Joyce ***@***.***>
Envoyé : dimanche 18 juillet 2021 03:17
À : sfu-dhil/wilde-schema
Cc : Colette Colligan; Author
Objet : Re: [sfu-dhil/wilde-schema] Collecting Reports (#1)


@ubermichael commented on this pull request.

________________________________

In docs/collecting_reports.md<https://github.com/sfu-dhil/wilde-schema/pull/1#discussion_r671762804>:

> @@ -0,0 +1,35 @@

+# Collecting Reports

+

+## Overview

+Newspapers around the world formatted news articles very differently, and digitization technologies have a variety of approaches to article segmentation. Together, these factors raise a big question: what is a news article? The Wilde Trials International News Archive has developed some criteria for determining what constitutes a news article and gets transcribed as an individual file.

+

+## Search Protocols

+Collect all relevant articles on the Wilde trials and aftermath between April 1 1895 to Dec 31 1895. When browsing a newspaper for articles, make sure to examine each issue printed within the period. When searching a newspaper database for articles, use the search term “Oscar Wilde.” Then try “Wilde,” “Queensberry,” and “Lord Alfred Douglas” to see if these terms catch more results.

+

+## Collection Protocols

+To determine what constitutes an article, prioritize evidence from the newspaper over the database’s article segmentation. In other words, does the newspaper represent it as an individual article? (eg. Does it have a headline; is it separated from other articles?). In most cases, it should be easy to figure out what is an individual news article.

+

+If there are a few different news articles published on the same day which appear in different parts of the newspaper, each of these news articles is considered a distinctive news article and gets transcribed as an individual file.

+

+If there is one long column, with multiple news items, including items pertaining to the Wilde trials, transcribe the relevant news items, while omitting the non-relevant news items. Use [xyz] to indicate if there is text that you have excerpted in the middle of the selected news artcle.



Is [xyz] the correct substitution? I thought we were using <p class='omitted'></p> or something similar.

**Comment by ccolliga** [comment]
On the transcription doc, this is what I wrote out for the quotations. My sense is that there might be too much variety for the schema to be set for correct usage.

For English-language newspapers, use straight quotation marks (" "). For French-language newspapers, use guillemets (« »). For German-language newspapers, use („ “). For Spanish and Italian-language newspapers, follow the practice of the newspapers, which may use quotation marks or guillemets.

For ampersands (&) in an xml document, including in a URL, type "&amp". (Here in markdown, I could not add the ";" for if I did, the ampersand was displayed rather than "&amp;"

**Pull request merged**